api/src/main/java/jakarta/servlet/http/HttpSessionContext.java

```diff
@@ -55,5 +58,7 @@
      *
      */
     @Deprecated
-    public Enumeration<String> getIds();
+    default public Enumeration<String> getIds() {
```

Comment:
Well this is awesome, we have contradictory Javadoc on the method (the first part says empty enumeration, the @return says null).

I think an empty enumeration would be safer here.

Reply:
Done and updated the Javadoc as well.